src/utilities/confusedMessageGenerator.js

```diff
@@ -0,0 +1,17 @@
+function confusedMessageGenerator(possibleLanguages, reactionEmojies) {
+  let questions = [];
+
+  possibleLanguages.forEach((language, index) => {
+    questions.push(
+      `Is it \`${language}\`? (If it is \`${language}\`, then react to the unformatted message with a ${reactionEmojies[index]} emoji)`
+    );
+  });
```

Comment:
This can be done using a `.map` call rather than a `.forEach`

Reply:
Replaced `forEach` with `map`.